dbmi-bgm/cgap-portal
# Auth0 Symlink Bugfix

**Changelog:**
- Tweak to Webpack config to prevent loading of Auth0-Lock from SPC peer deps (doesn't seem to work in Webpack 5 for whatever reason)


**Steps to test this:**
*Note: Would be good to do this two times. First, with master to double check the bug is occurring, then second with this branch to ensure this actually fixes it.*
1. Pull down and checkout latest master on both SPC and CGAP. (On second test, checkout this branch on CGAP)
2. Delete node_modules from both SPC and CGAP local repos
3. `Make clean` && `make build` on CGAP
4. `npm ci` on SPC (can be done at same time as step 3)
5. `npm link` on SPC (probably good to wait until after 3 + 4 complete)
6. `npm link @hms-dbmi-bgm/shared-portal-components` on CGAP
7. Start CGAP as normal (`make deploy1`, `make deploy2`, etc)
8. Go to home page, and check if login button and lock are active (Should be broken on master, and work with this branch)
